paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionViewAction.kt

```diff
@@ -3,4 +3,5 @@ package com.stripe.android.paymentsheet.paymentdatacollection.cvcrecollection
 internal sealed interface CvcRecollectionViewAction {
     data class OnConfirmPressed(val cvc: String) : CvcRecollectionViewAction
     object OnBackPressed : CvcRecollectionViewAction
+    data class CvcCompletionChanged(val completion: CvcCompletionState) : CvcRecollectionViewAction
```

Comment:
We can't reuse this for the flow controller variant because it has different requirements. Unlike `PaymentSheetScreen` which handles the confirm/buy button state, `FlowController` lets the cvc recollection screen manage the UI state and then receives the cvc as an ActivityResult

Reply:
I'll go with your `CvcValueChanged` suggestion though. It's simpler than what I currently have